### ISSUE TITLE
1.12 Branch

### ISSUE THREAD

**Issue by Brycey92** [EXTER7/OreDictionaryConverter#16]
@EXTER7 Can you create a 1.12 branch so I can make a PR for my work to port this mod to MC 1.12? Thanks!

**Comment by Brycey92** [comment]
Any progress on making this branch?

**Comment by HurtsAsHell** [comment]
have you already finished the porting? @Brycey92

**Comment by Brycey92** [comment]
@HurtsAsHell yes, I'm running the mod on my 1.12 pack and server right now, and I have been for quite some time. I just wanted my port on the official repo/CurseForge so people can more easily find and use it.

**Comment by minakitty** [comment]
@Brycey92 Do you happen to have a pre-compiled jar available from your pack?  I was considering trying to use it, or something like it, in the server I'm reviving (doing the conversion at worldgen would be preferable, but I never seem to be able to get it all)
